packages/desktop/views/login/views/SelectProfileView.svelte

```diff
@@ -49,13 +48,9 @@
         {#each $profiles as profile}
             <div class="mx-7 mb-8">
                 <Profile
+                    {profile}
                     bgColor="blue"
                     onClick={onContinueClick}
-                    name={profile.name}
-                    id={profile.id}
-                    isDeveloper={profile.isDeveloperProfile}
-                    networkType={profile?.networkType ?? NetworkType.Devnet}
-                    networkProtocol={profile?.networkProtocol ?? NetworkProtocol.IOTA}
                     isLedgerProfile={profile?.type === ProfileType.Ledger}
                     updateRequired={profile?.type === ProfileType.Software &&
                         !isStrongholdUpdated(profile) &&
```

Comment:
These props could also be covered just by passing the profile to the `Profile` component right?

Reply:
Yeah its a suggestion for mobile I changed it on desktop but this is just a patch for mobile, I wasn't going to refactor it too much when I cant test.